src/libsync/creds/oauth.cpp

```diff
@@ -514,6 +514,12 @@ bool isUrlValid(const QUrl &url)
 {
     qCDebug(lcOauth()) << "Checking URL for validity:" << url;
 
+    // we have hardcoded the oauthOverrideAuth
+    const auto overrideUrl = Theme::instance()->oauthOverrideAuthUrl();
+    if (!overrideUrl.first.isEmpty()) {
+        return QUrl::fromUserInput(overrideUrl.first).matches(url, QUrl::RemoveQuery);
```

Comment:
I guess the overrideUrl is a list and you take the first entry and match it to the type "url" ... now if that is oc-auth:// is that recognized as URL?